app/javascript/app/components/province/province-styles.scss

```diff
@@ -0,0 +1,58 @@
+@import '~styles/settings.scss';
+@import '~styles/layout.scss';
+
+.provinceContainer {
+  @include columns();
+
+  justify-content: center;
+
+  @media #{$tablet-landscape} {
+    @include columns($columns: (6, 6));
```

Comment:
The second column seems to be 5 so the text is aligned with the title
![image](https://user-images.githubusercontent.com/9701591/48347471-9e3f0c80-e67e-11e8-827a-c8bfdfa1b9dc.png)